sdk/servicebus/service-bus/changelog.md

```diff
@@ -1,3 +1,20 @@
+# 2019-05-15 1.0.0
+
+- Pass value for `associated-link-name` in the application properties when making a request over the
+$management link. This is used to carry out optimizations by the service on the server side.
+- User agent string which is passed as a AMQP connection property is updated to follow the new standard.
+For example: `azsdk-js-azureservicebus/1.0.0/(NODE-VERSION v10.15.0) Windows_NT 10.0.17763`
+- Avoid type checks on the message before sending it, as `rhea` throws type errors while encoding the message.
+- `receiveMessages()` now returns rejected promise if network connection is lost instead of getting stuck
+forever waiting for the `drain` event to be fired by rhea
+- Empty string is a valid sessionId as per Service Bus, therefore allow it to be used when receiving messages.
+- When Service Bus does not acknowledge a message settlement/disposition request in time, throw an error
+instead of returning a resolved promise that misleads into assuming a successful operation.
```

Comment:
I dont know what you are trying to say here 😟

Reply:
Sorry, I forget not everyone uses VIM. I just mean, could remove "that misleads into assuming a successful operation" since it's not super useful to understand the changes.

(If you're curious, `s` = substitute, and after the slash is a regexp to match, and after the next slash is the replacement text, which in this case is empty so means replace with nothing).

Reply:
got it, fixed